corehq/apps/events/tests/test_models.py

```diff
@@ -317,6 +318,43 @@ def example_config(self):
             config.delete()
 
 
+class EventCaseTests(TestCase):
+
+    def setUp(self):
+        today = datetime.utcnow().date()
+        self.event = Event(
+            name='Test Event',
+            domain=DOMAIN,
+            start_date=today,
+            end_date=today,
+            attendance_target=0,
+        )
+        self.event.save()
+
+    def tearDown(self):
+        try:
+            self.event.delete()
```

Comment:
Why is this put into a try/except block? :thinking:

Reply:
Two of the tests delete the event. If you try to delete a deleted event it raises an exception.

Reply:
Ah okay, that makes sense. In that case, you could add the specific Exception type you're looking to catch to make it clear why the except is there